### ISSUE TITLE
Need to delete/close over 50 empty issues (noise/pollution)

### ISSUE THREAD

**Issue by bgadrian** [golang/tour#507]
Hello,

I have identified around 50 issues that most likely were added by mistake, with the standard message , title and no other info. I would list them here to facilitate easy removal for an admin, hopefully.
The problem is that the report button si too visible, and the people that visit the website are not familiar with reporting bug process.
The result is that is very hard to find the real issues/bugs because the list is too poluted

Part 1/2:
https://github.com/golang/tour/issues/510
https://github.com/golang/tour/issues/500
https://github.com/golang/tour/issues/496
https://github.com/golang/tour/issues/492
https://github.com/golang/tour/issues/491
https://github.com/golang/tour/issues/484
https://github.com/golang/tour/issues/475
https://github.com/golang/tour/issues/467
https://github.com/golang/tour/issues/465
https://github.com/golang/tour/issues/459
https://github.com/golang/tour/issues/456
https://github.com/golang/tour/issues/446
https://github.com/golang/tour/issues/440
https://github.com/golang/tour/issues/434
https://github.com/golang/tour/issues/432
https://github.com/golang/tour/issues/427
https://github.com/golang/tour/issues/424
https://github.com/golang/tour/issues/421
https://github.com/golang/tour/issues/418
https://github.com/golang/tour/issues/415
https://github.com/golang/tour/issues/407
https://github.com/golang/tour/issues/405
https://github.com/golang/tour/issues/402
https://github.com/golang/tour/issues/380
https://github.com/golang/tour/issues/378
https://github.com/golang/tour/issues/377
https://github.com/golang/tour/issues/368
https://github.com/golang/tour/issues/356
https://github.com/golang/tour/issues/352
https://github.com/golang/tour/issues/351
https://github.com/golang/tour/issues/347
https://github.com/golang/tour/issues/346
https://github.com/golang/tour/issues/345

**Comment by ALTree** [comment]
I just closed all the issues linked here. Thank you for doing this!

**Comment by bgadrian** [comment]
Thank you @ALTree , I continued the search (from 345 backwards) and I have found these similar invalid issues (no descriptions)

https://github.com/golang/tour/issues/350
https://github.com/golang/tour/issues/336
https://github.com/golang/tour/issues/335
https://github.com/golang/tour/issues/334
https://github.com/golang/tour/issues/333
https://github.com/golang/tour/issues/326
https://github.com/golang/tour/issues/325
https://github.com/golang/tour/issues/317
https://github.com/golang/tour/issues/244
https://github.com/golang/tour/issues/243
https://github.com/golang/tour/issues/103

I will continue to group the rest of the tasks (in other issues) there are some common problems with duplicate issues.

**Comment by bgadrian** [comment]
I went trough most of the open issues and I have found the following tickets that can be closed (in my opinion) for several reasons (I grouped them): 

**The author haven't returned with more feedback (in more then 1year), it was a question that was answered OR it was an issue because it didn't read the notes.**

https://github.com/golang/tour/issues/46
https://github.com/golang/tour/issues/47
https://github.com/golang/tour/issues/60
https://github.com/golang/tour/issues/72
https://github.com/golang/tour/issues/91
https://github.com/golang/tour/issues/113
https://github.com/golang/tour/issues/126
https://github.com/golang/tour/issues/129
https://github.com/golang/tour/issues/137
https://github.com/golang/tour/issues/229
https://github.com/golang/tour/issues/255
https://github.com/golang/tour/issues/384
https://github.com/golang/tour/issues/390
https://github.com/golang/tour/issues/429
https://github.com/golang/tour/issues/444
https://github.com/golang/tour/issues/493
https://github.com/golang/tour/issues/495
https://github.com/golang/tour/issues/509
https://github.com/golang/tour/issues/538

**Duplicates of #8 (go tour CLI is not working)**

https://github.com/golang/tour/issues/396
https://github.com/golang/tour/issues/116
https://github.com/golang/tour/issues/160
https://github.com/golang/tour/issues/153
#175


**"date is wrong" or "random is not working"**
https://github.com/golang/tour/issues/109
https://github.com/golang/tour/issues/131
https://github.com/golang/tour/issues/96

https://github.com/golang/tour/issues/267
https://github.com/golang/tour/issues/468


**Suggestion to the languages or general go questions, not related to /tour:** & they have answers.

https://github.com/golang/tour/issues/50
https://github.com/golang/tour/issues/72
https://github.com/golang/tour/issues/156
https://github.com/golang/tour/issues/226
https://github.com/golang/tour/issues/361
https://github.com/golang/tour/issues/463
https://github.com/golang/tour/issues/504

**Comment by ALTree** [comment]
Done the second batch. I left a couple open for a few issues where I believe we could do better.

**Comment by bgadrian** [comment]
I found a few other solved issues, I missed them in earlier runs.  

https://github.com/golang/tour/issues/337 - didn't read the notes
https://github.com/golang/tour/issues/298 - didn't read the notes
https://github.com/golang/tour/issues/290 - didn't read the notes
https://github.com/golang/tour/issues/404 - didn't read the notes (what defer does)
https://github.com/golang/tour/issues/454 - empty issue

Now that a lot of "noise" is gone, I can start finding duplicate issues and maybe do PRs to actually fix stuff.

**Comment by bgadrian** [comment]
Over 150 issues were closed I think, good job @ALTree ! 

If you want I could go over all the issues one more time and add here the duplicates, or you can close this issue if already done this. Thanks!

**Comment by ALTree** [comment]
I think we can close here, most of the obviously invalid issues are now closed. Many of the dups are about translation error, those are blocked on a decision on #550. Thanks!
